app.js

```diff
@@ -0,0 +1,91 @@
+/* jshint esversion:6 */
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+const port = 8080;
+
+let wordArr = [];
+let score = 0;
+
+let getBuzzArr;
```

Comment:
This variable name would me be think it is a function. get is generally applied to an action, give it a name that matches what it contains. Additionally, if this is going to be an array you might want to initialize it.